DataStructures/LinkedList/LinkedList.cpp

```diff
@@ -0,0 +1,160 @@
+#include "LinkedList.hpp"
+
+/*
+    Author:
+    Tucker J. Polomik (https://github.com/TheNergaL)
+    Name of algorithm: 
+    Singly Linked-list
+    Brief description:
+    List comprised of nodes which hold a value
+    and point to the next value element in the list.
+    If there exists no next element, it points to null.
+
+    Time complexity:
+    Element search/access: O(N)
+    Data insertion: O(1)
+    Data deletion: O(1)
+    where N is the number of nodes (elements) in the list.
+
+    Space complexity:
+    O(n), where n is the number of nodes (elements) in a list
+*/
+
+template <class T>
+LinkedList<T>::LinkedList() {    //constructor
+    head = nullptr;
+    last = nullptr;
+}
+
+template <class T>
+LinkedList<T>::~LinkedList() {    //destructor
+    Node<T>* temp_node = head;
+    while(temp_node != nullptr) {
+        temp_node = temp_node->nextnode;
+        delete(head);
+        head = temp_node;
+    }
+}
+
+template <class T>
+bool LinkedList<T>::isEmpty() {
+    return (head == nullptr);
+}
+
+template <class T>
+int LinkedList<T>::size() const {
+    if (head == nullptr) { return 0; }
+        Node<T>* temp_node = head;
+        int nodeWalk = 0;

```

Comment:
Please shift the return statement to the next line, without braces.
Also, the following 2 lines have an extra level of indentation.